src/View/Helper/FormHelper.php

```diff
@@ -601,12 +622,19 @@ public function secure(array $fields = [], array $secureAttributes = [])
             'value' => $tokenData['unlocked'],
         ]);
         $out .= $this->hidden('_Token.unlocked', $tokenUnlocked);
+        if (!empty($tokenData['optional'])) {
+            $tokenOptional = array_merge($secureAttributes, [
```

Comment:
I was following the code examples in the lines before and after, which all use array_merge.